index.html

```diff
@@ -0,0 +1,12 @@
+<!-- index.html -->
```

Comment:
prepare this html in a proper way :) add doctype, lang attribute, remove comments